tests/InputFiles/GeneralizedHarmonic/GaugeWave1D.yaml

```diff
@@ -34,7 +34,7 @@ PhaseChangeAndTriggers:
           WallclockHours: None
 
 InitialData: &InitialData
-  GaugeWave:
+  WrappedGr(GaugeWave):
```

Comment:
Still reviewing this PR, but a quick note on this:  As a person not very familiar with GH system, the fact that I should use `WrappedGr(GaugeWage)` for running `GaugeWage` feels a bit unnatural to me. 

If this is only about avoiding confusions in input files, can we leave GH input files still use the unwrapped version (`GaugeWave`) of specifier, and only use `WrappedGr` keyword for GhValenciaDivClean input files? Maybe by template specialization or just overloading `name()` function. Not sure how hard that would be.
```

Reply:
There are a couple reasons why this needs to happen; I agree this is not ideal.  Maybe switching the name to `GeneralizedHarmonic(...`  is good enough for the near term?  The problem is that all of these things are the same under the hood; i.e.  the Ghmhd initial data is the same thing as the Gh initial data.   A change to the Ghmhd option tag may have to change with runtime initial data anyway I think;  I'm happy to file an issue to find a better solution for this once the dust has settled.